resolverway/views.py

```diff
@@ -106,16 +106,23 @@ def set_user_info(self, request):
             self.real_ip = request.headers.get('x-real-ip', None)
         session = request.cookies.get('session', None)
         if session:
-            account = redis_db.get(session)
+            try:
+                account = redis_db.get(name=current_app.config['REDIS_NAME_PREFIX']+session)
+            except:
+                account = None
+                current_app.logger.error('exception on getting user info from cache with session=%s' % (session))
             if account:
                 current_app.logger.info('getting user info from cache with session=%s' % (session))
                 # account is saved to cache as string, turned it back to dict
                 account = ast.literal_eval(account)
             else:
                 account = self.get_user_info_from_adsws(session)
                 if account:
-                    # save it to cache
-                    redis_db.set(session, account)
+                    try:
+                        # save it to cache for a week
+                        redis_db.set(name=current_app.config['REDIS_NAME_PREFIX']+session, value=account, ex=604800)
```

Comment:
Could we make the expiration value a config variable? Something like `current_app.config['REDIS_EXPIRATION_TIME']`, this way we can control that from the config file when deploying the microservice and we will never need to touch the code.